assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js

```diff
@@ -109,4 +123,10 @@ export default createWidget("discourse-reactions-list-emoji", {
       }
     });
   },
+
+  _allowHover() {
```

Comment:
Yes this is not ideal, but there is seemingly no way to get rid of `(hover: none)` in chrome headless. Maybe in time we could globally set `window.isSystemTest` in system tests, like how we have `isTesting()` available from ember

Reply:
> Maybe in time we could globally set `window.isSystemTest` [...]

👍